configure

```diff
@@ -38,6 +39,8 @@ run_erlang() {
 
 COUCHDB_USER="$(whoami 2>/dev/null || echo couchdb)"
 SM_VSN=${SM_VSN:-"91"}
+CLOUSEAU_VSN=${CLOUSEAU_VSN:-"2.22.0"}
+CLOUSEAU_DIR="$(pwd)"/clouseau
```

Comment:
Yes, the top-level `nouveau` directory inspired me.  Also, I wanted to make Clouseau, as its sibling, look similar therefore easier for the developers to understand how it works.  Mind that `clouseau` will not hold any source files but "executable" files (JARs) and configuration files.  Thus it would not make a good addition to `src`, I believe.